pkgs/top-level/all-packages.nix

```
@@ -21027,6 +21027,8 @@ with pkgs;
 
   bird = callPackage ../servers/bird { };
 
+  bloat = callPackage ../servers/bloat { };
```

The description says that this is a client. So why `servers`?

```suggestion
  bloat = callPackage ../applications/networking/instant-messengers/bloat { };
```
would be better. Other Mastodon clients are also in that catergory.

You'd still have to move the `default.nix`.

---

Well it's kind of both! It is a pleroma/mastodon client, but it runs a web server that serves the frontend, and it's mostly set up by server operators rather than by users themselves

---

The client may use a web server but it's still a client and not serving other users, right? So it shouldn't be in the server category.

---

> The client may use a web server but it's still a client and not serving other users, right?

It's made to serve multiple users. eg, https://bloat.freesoftwareextremist.com/ is meant to be used by any user of freesoftwareextremist.com (or any other mastodon / pleroma instance for that matter)

---

So It'd be nice to have a module to enable the server, right?

---

Yes I do plan to write one, just not in this pr